### ISSUE TITLE
Not compatible with ActiveStorage

### ISSUE THREAD

**Issue by ledermann** [jorgemanrubia/turbolinks_render#9]
Since v0.9.9/v0.9.10, the gem seems not to be compatible with ActiveStorage in development mode. With v0.9.1 it works fine.

As far as I can tell the middleware prevents ActiveStorage to access disk files. Example:

A NoMethodError is raised while accessing a file, e.g. `http://my-app.test/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDRG9JYTJWNVNTSWRhRmc1ZWpSclFYazJlV1pDYXpNNVJWUlpWSFZvV1VGUUJqb0dSVlE2RUdScGMzQnZjMmwwYVc5dVNTSS9hVzVzYVc1bE95Qm1hV3hsYm1GdFpUMGlRbXgxYldWdUxuQnVaeUk3SUdacGJHVnVZVzFsS2oxVlZFWXRPQ2NuUW14MWJXVnVMbkJ1WndZN0JsUTZFV052Ym5SbGJuUmZkSGx3WlVraURtbHRZV2RsTDNCdVp3WTdCbFE9IiwiZXhwIjoiMjAxOS0wMi0yNVQxNDoxNDo0Ny40NzdaIiwicHVyIjoiYmxvYl9rZXkifX0=--03972963f31948d4b694a5425b7b8e3191093f4c/example.png?content_type=image%2Fpng&disposition=inline%3B+filename%3D%22example.png%22%3B+filename%2A%3DUTF-8%27%27example.png`.

Here is the stack trace:

```
Puma caught this error: undefined method `body' for #<Rack::File::Iterator:0x00007f8f675431b8> (NoMethodError)
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/http/response.rb:303:in `body'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/http/response.rb:485:in `body'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-2.0.6/lib/rack/body_proxy.rb:41:in `method_missing'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-2.0.6/lib/rack/body_proxy.rb:41:in `method_missing'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/turbolinks_render-0.9.10/lib/turbolinks_render/middleware.rb:15:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/rack/logger.rb:38:in `call_app'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/rack/logger.rb:26:in `block in call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/tagged_logging.rb:71:in `block in tagged'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/tagged_logging.rb:28:in `tagged'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/tagged_logging.rb:71:in `tagged'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/rack/logger.rb:26:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/sprockets-rails-3.2.1/lib/sprockets/rails/quiet_assets.rb:13:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/request_store-1.4.1/lib/request_store/middleware.rb:19:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/middleware/request_id.rb:27:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-2.0.6/lib/rack/method_override.rb:22:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-2.0.6/lib/rack/runtime.rb:22:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/activesupport-5.2.2/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/middleware/executor.rb:14:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/actionpack-5.2.2/lib/action_dispatch/middleware/static.rb:127:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-2.0.6/lib/rack/sendfile.rb:111:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/webpacker-4.0.0.rc.7/lib/webpacker/dev_server_proxy.rb:29:in `perform_request'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-proxy-0.6.5/lib/rack/proxy.rb:57:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/railties-5.2.2/lib/rails/engine.rb:524:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rack-canonical-host-0.2.3/lib/rack/canonical_host.rb:19:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/puma-3.12.0/lib/puma/configuration.rb:225:in `call'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/puma-3.12.0/lib/puma/server.rb:658:in `handle_request'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/puma-3.12.0/lib/puma/server.rb:472:in `process_client'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/puma-3.12.0/lib/puma/server.rb:332:in `block in run'
/Users/ledermann/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/puma-3.12.0/lib/puma/thread_pool.rb:133:in `block in spawn_thread'
```

**Comment by jorgemanrubia** [comment]
HI @ledermann,

Yeah I changed a few things internally in recent versions. I'll fix this asap. Thanks for reporting it. 

Cheers

**Comment by jorgemanrubia** [comment]
@ledermann I just released `0.9.11` that should fix this and also improves performance a lot.

Thanks

**Comment by ledermann** [comment]
Works fine, thank you for your great work on this project!
